docs/source/recipes/appextras.md

```diff
@@ -1,14 +1,18 @@
 ---
 myst:
   html_meta:
-    "description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
-    "property=og:description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
-    "property=og:title": "AppExtras component"
-    "keywords": "Volto, Plone, frontend, React, app extra component"
+    'description': 'The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code.'
+    'property=og:description': 'The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code.'
+    'property=og:title': 'AppExtras component'
+    'keywords': 'Volto, Plone, frontend, React, app extra component'
 ---
 
 # AppExtras component
 
+```{versionchanged}17.x.x and 16.x.x
```

Comment:
We need explicit version numbers. Let's use 17.11.6 and 16.30.3, based on https://www.npmjs.com/package/@plone/volto?activeTab=versions. @sneridagh will need to verify upon release.

````suggestion
```{versionchanged} 17.11.6 and 16.30.3
````